Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm

```diff
@@ -1240,6 +1240,8 @@ static void activateSessionCleanup(NetworkSessionCocoa& session, const NetworkSe
 
     NSURLSessionConfiguration *configuration = configurationForSessionID(m_sessionID, isParentProcessAFullWebBrowser(networkProcess));
 
+    m_blobRegistry.setFileDirectory(FileSystem::createTemporaryDirectory(@"BlobRegistryFiles"));
```

Comment:
Maybe the idea is that the folder we use is randomized per session, and then the files within it are sequentially numbered? Still seems slightly simpler to me to use the OS API for randomizing each file path. But maybe that's not how we usually do it.

Reply:
Right, that is the idea, this seems simple enough and is not limited to XXXXX concurrent files like mkstemp.
I could also test whether we could use a single file per NetworkProcess, maybe reusing the same file path is ok given we are deleting the file as soon as we are using it.

Reply:
I think this solution is fine as-is, and it seems to match the idiom in WebKit. Just a bit surprising to me as a first time reader.